mono/utils/mono-dl-fallback.h

```diff
@@ -10,10 +10,13 @@
 MONO_BEGIN_DECLS
 
 enum {
-	MONO_DL_EAGER = 0,
-	MONO_DL_LAZY  = 1,
-	MONO_DL_LOCAL = 2,
-	MONO_DL_MASK  = 3
+	MONO_DL_EAGER  = 0,
+	MONO_DL_LAZY   = 1,
+	// If MONO_DL_LOCAL is set, it will trump MONO_DL_GLOBAL
+	MONO_DL_LOCAL  = 2,
+	// This is unused internally and no longer a full mask, given the introduction of MONO_DL_GLOBAL. Avoid.
```

Comment:
I find “this” in comments annoyingly ambiguous. Can you repeat the name instead?

Reply:
I agree and fixed it locally a couple days ago, just haven't pushed.